src/lib/Suite.ts

```diff
@@ -346,6 +346,16 @@ export default class Suite implements SuiteProperties {
     );
   }
 
+  /**
+   * Explicity reset the suite so it may run again
+   */
+  reset() {
+    this._remote = undefined;
+    // Remove implicit reset from run() (in case it's relied upon)
```

Comment:
Since this is going in v5, go ahead and actually remove the implicit reset code from Suite. Move the TODO about canceling suite runs here.